src/devices/vdml_adi.c

```diff
@@ -128,3 +128,23 @@ adi_potentiometer_t adi_potentiometer_type_init(uint8_t port, adi_potentiometer_
 double adi_potentiometer_get_angle(adi_potentiometer_t potentiometer) {
 	return ext_adi_potentiometer_get_angle((ext_adi_potentiometer_t)potentiometer);
 }
+
+adi_led_t adi_led_init(uint8_t port) {
+	return (adi_led_t)ext_adi_led_init(INTERNAL_ADI_PORT, port);
+}
+
+int32_t adi_led_set_state(adi_led_t led, bool value) {
```

Comment:
can we change this to turn on/turn off? This would be more intuitive to the user

(or turn_on(adi_led_t led) and turn_off(adi_led_t led) and turn(adi_led_t led, bool on) )